BladeportBinaryTreeManager.Business/UserBL.cs

```diff
@@ -25,5 +25,20 @@ public List<UserDTO> GetUserList()
                 JoinDate = x.JOINDATE
             }).ToList();
         }
+
+        void IUserBL.AddUser(UserDTO user)
+        {
+            _ctx.AddUser(user);            
+        }
+
+        void IUserBL.DeleteUser(UserDTO user)
+        {
+            _ctx.DeleteUser(user);
+        }
+
+        void IUserBL.EditUser(UserDTO user)
+        {
+            _ctx.EditUser(user);
+        }
```

Comment:
CRUD operations should return a value e.g. boolean to indicate Success or Failure. This value will be used by the User Interface to react accordingly